sh.shuriken.Eeman.json

```
@@ -0,0 +1,100 @@
+{
+    "id" : "sh.shuriken.Eeman",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "eeman",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=session-bus"
```

```suggestion
        "--socket=wayland"
```

Shouldn't be needed anymore. This is needed for a test build to happen

---

Done however the app will fail to run because I think changing app ID causes some backend issues.. not able to deregister the app from org.Gtk.Application.